### ISSUE TITLE
Marker label and marker click tolerance

### ISSUE THREAD

**Issue by MaticDiba** [Mapsui/Mapsui#480]
I have two questions regarding markers on the map.

I am adding markers to the map with as a features with text label:

```c#
var features = new Features();
foreach (var point in points)
            {
                var feature = new Feature
                {
                    Geometry = point.Location, // type of Point
                    ["Type"] = "Point"
                };
                feature.Styles.Add(new LabelStyle { Text = point.Name});
                features.Add(feature);
            }
```
After that I add features to the layer:
```c#
Layer layer = new Layer("pointlayer")
            {
                Name = layername,
                DataSource = new MemoryProvider(features),
                
                Style = new SymbolStyle
                {
                    SymbolScale = 0.4,
                    Fill = new Brush(Mapsui.Styles.Color.Red),
                    Outline = { Color = Mapsui.Styles.Color.Black, Width = 2},
                    SymbolOffset = new Offset(0, 40),
                }
            };
```
I add layer to InfoLayers. So now, I can catch the click/tap on the point.

My first question: Is it possible to hide labels until Resolution is lower then x? And vice versa.

My second question: I want markers to be small. But then, they are hard to tap/click on. Is it possible to 'widen' clickable area?

**Comment by Wil69** [comment]
Hi I try to use MinVisible and MaxVisible : 
feature.Styles.Add(new LabelStyle { Text = point.Name, MinVisible = 0, MaxVisible = 4 });
And this is not working

But when I add features to the layer and I add MinVisible and MaxVisible that works well

@MaticDiba & @pauldendulk Do you see the same thing ?

**Comment by MaticDiba** [comment]
First, thanks for the answer above.

But I also have the same problem as @Wil69.  I tried this before and throught that this is not the correct solution.

Regarding second item. I guess adding transparent outline to the symbol, wouldn't work the same as with the image?

**Comment by charlenni** [comment]
Be careful: theResolution isn't in Google zoom levels (2^n). So check the possible resolutions for your map. Normally Resolution for maps in Mapsui (using OSM and so on) is in meters per pixel at equator. So 4 is a very huge zoom factor.

**Comment by MaticDiba** [comment]
Yes, I chose the right resolution. As Wil69 said, it does work with layer (if I set the style to the layer) and it shows the markers at the right resolution. But if I set that style to the feature, it doesn't work.
example:
```c#
var feature = new Feature
                {
                    Geometry = point.Location,
                    ["Type"] = "point"
                };
                feature.Styles.Add(new LabelStyle { Text = point.Name, MaxVisible =x });
```

**Comment by Wil69** [comment]
Hi @charlenni , I try with the same value for both cases ....

**Comment by pauldendulk** [comment]
@Wil69 The MinVisible and MaxVisible filtering on features is indeed not applied. I added this in a recent commit.

**Comment by pauldendulk** [comment]
@MaticDiba I assume a wide transparent outline would work. Outlines do not apply to bitmap symbols though.

**Comment by MaticDiba** [comment]
Great for the fix with new commit. Although few days ago I tried to go to version 1.4.4 from 1.4.2 but the map with Osm layer was responding really slowly. I didn't have much time to investigate this issue, so I just reverted back to 1.4.2.

**Comment by pauldendulk** [comment]
Yes, this is different issue that needs to be fixed. I now do not run on the ui thread when not needed, this might have an effect.

**Comment by pauldendulk** [comment]
This should be fixed in 1.4.7.
